packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/CameraPlugin.m

```diff
@@ -224,8 +224,7 @@ - (void)initializeCamera:(NSInteger)cameraId
 - (void)startImageStreamWithCompletion:(nonnull void (^)(FlutterError *_Nullable))completion {
   __weak typeof(self) weakSelf = self;
   dispatch_async(self.captureSessionQueue, ^{
-    [weakSelf.camera startImageStreamWithMessenger:weakSelf.messenger];
-    completion(nil);
+    [weakSelf.camera startImageStreamWithMessenger:weakSelf.messenger withCompletion:completion];
```

Comment:
This needs to be converted to a `strongSelf`, and a codepath added that calls `completion(nil)` if `strongSelf` is `nil`. As written, this will never call `completion` if `weakSelf` is `nil`, which violates the API contract of platform channel completion handlers.